stimuli_ci_generation/src/server/main.py

```diff
@@ -60,7 +60,12 @@ def index():
             return f"Invalid file identifier {file_identifier}", 204
         identifier, file_name = file_identifier_parts
         file_type = file_name.split(".")[-1]
-        participant_id, experiment_id = identifier.split("-")
+
+        try:
+            participant_id, experiment_id = identifier.split("-")
+        except ValueError:
+            return f"Failed to parse identifer {identifier}", 204
```

Comment:
This should not be 204 right?

Reply:
it's 204 to ack the pubsub, or else they don't get acked. The only two status codes we use is 200 for OK and 204 for failure